dojo/models.py

```diff
@@ -378,6 +392,7 @@ class UserContactInfo(models.Model):
     slack_username = models.CharField(blank=True, null=True, max_length=150, help_text="Email address associated with your slack account", verbose_name="Slack Email Address")
     slack_user_id = models.CharField(blank=True, null=True, max_length=25)
     block_execution = models.BooleanField(default=False, help_text="Instead of async deduping a finding the findings will be deduped synchronously and will 'block' the user until completion.")
+    force_password_reset = models.BooleanField(default=False, help_text='Forces this user to reset password upon first sign in.')
```

Comment:
@blakeaowens Might be a bit picky, but shouldn't the help_text say "Forces this user to reset password at next sign in."? The flag can be set for existing users as well, not only for new users.

Reply:
Sure, I will make that change!